usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -1532,6 +1532,9 @@ static CK_RV rawkey_2_blob(STDLL_TokData_t * tokdata, SESSION * sess,
                              &cslen, (uint64_t) ep11_data->target_list);
     RETRY_END(rc, tokdata, sess)
 
+    if (rc == 0x8001000d)
```

Comment:
Is there no ep11 header that has a readable define for this . If not, should we add such a define?

Reply:
No unfortunately not.... EP11 defines lost of private return codes. Just adding this one seems to be strange.

Reply:
okay